src/app/archive/archive_lib/load_data.ml

```diff
@@ -774,9 +780,8 @@ let get_account_accessed ~pool (account : Processor.Accounts_accessed.t) :
             }
             : Mina_base.Zkapp_account.t ))
   in
-  let%bind zkapp_uri =
-    query_db ~f:(fun db -> Processor.Zkapp_uri.load db zkapp_uri_id)
```

Comment:
can we keep it this way until the field is moved? or maybe create an issue and we can prioritize to make the change in a follow up PR

Reply:
since the zkapp account is an option, but the URI is not an option, how about:
```
  let%bind zkapp_uri =
    Option.value_map zkapp_db ~default:(return "https://dummy.com")
      ~f:(fun zkapp -> query_db ~f:(fun db -> Processor.Zkapp_uri.load db zkapp.zkapp_uri_id))
  in
```